lib/nostrum/struct/sticker/pack.ex

```diff
@@ -0,0 +1,93 @@
+defmodule Nostrum.Struct.Sticker.Pack do
+  @moduledoc """
+  Represents an in-built sticker pack on Discord
```

Comment:
Is `in-built` correct? You're the native English speaker, and it reads fine, but I feel `Represents a built-in sticker pack` would sound more natural, or am I going crazy?

Reply:
I'm actually going to sub this for "Represents a platform-curated sticker pack on Discord"